basic_example_scenes.ipynb

```diff
@@ -142,27 +142,27 @@
     "        VGroup(title, basel).arrange(DOWN)\n",
     "        self.play(\n",
     "            Write(title),\n",
-    "            FadeInFrom(basel, UP),\n",
+    "            FadeIn(basel, shift = UP),\n",
     "        )\n",
     "        self.wait()\n",
     "\n",
     "        transform_title = Tex(\"That was a transform\")\n",
     "        transform_title.to_corner(UP + LEFT)\n",
     "        self.play(\n",
     "            Transform(title, transform_title),\n",
-    "            LaggedStart(*[FadeOutAndShift(obj, direction=DOWN) for obj in basel]),\n",
+    "            LaggedStart(*[FadeOut(obj, shift=DOWN) for obj in basel]),\n",
     "        )\n",
     "        self.wait()\n",
     "\n",
-    "        grid = NumberPlane()\n",
+    "        grid = NumberPlane(x_range=(- 10, 10, 1), y_range=(- 6.0, 6.0, 1))\n",
```

Comment:
I don't see any difference when you added the `x_range` and `y_range` so you could just remove it.

Reply:
I got the following difference without and with the range changes.
Old
![old](https://user-images.githubusercontent.com/76829691/120363854-9bf33b00-c2da-11eb-88f3-b82e795a91b2.png)
New
![new](https://user-images.githubusercontent.com/76829691/120363872-a1508580-c2da-11eb-9729-7cba9d2667aa.png)

Reply:
Ahh, I see. But, I would like to have consistency in the Jupyter Notebook examples and the Example Gallery in the documentations so you could either remove it here or change it over there.

Reply:
Thanks a lot for your contribution, @PaulCMurdoch !
I think that having complete redundancy at both places (Jupyter Notebook examples and the Example Gallery)
would be nice, but not necessary.
As that would include extra work, I think it is enough to be pragmatic and change it only here.
If you have time and are motivated, feel free to make an extra pr to the Example Gallery, but again, you are not obligated to anything :)